apps/condo/domains/ticket/access/TicketReportService.js

```diff
@@ -5,6 +5,14 @@ async function canReadTicketReportWidgetData ({ authentication: { item: user } }
     return { organization: { employees_some: { user: { id: user.id }, isBlocked: false, deletedAt: null } } }
 }
 
+async function canReadTicketReportAnalyticsData ({ authentication: { item: user } }) {
+    if (!user) return false
+    if (user.isAdmin) return true
+
+    return { organization: { employees_some: { user: { id: user.id, isBlocked: false, deletedAt: null } } } }
+}
+
 module.exports = {
     canReadTicketReportWidgetData,
+    canReadTicketReportAnalyticsData,
```

Comment:
Every service should’ve have access. Check create service until :)

Reply:
@pahaz I don't fully understand what this is about. TicketReportService now contains 2 queries, so I place access in existed file

Reply:
Could you please use `createservice` command?

Reply:
Fixed